database/migrations/2020_05_04_150000_create_bookings_table.php

```diff
@@ -16,22 +16,21 @@ public function up()
         Schema::create('bookings', function (Blueprint $table) {
             $table->id();
             $table->string('slot_number');
+            $table->foreignIdFor(app('user'));
+            $table->morphs('order'); // Can this be $table->foreignIdFor(app('order'));
```

Comment:
@sl0wik Can we also use the foreign relationship for Order?

Reply:
It won't work as far I know because morph would be order_type and order_id (so other order models can be used).

Reply:
What are the order types? Why wouldn't the order contain its own order type?

Reply:
Do you mean Cart & Order as the different types?

Reply:
We are renaming it to a normal foreign id to avoid confusion with the way morph working. Using morph you can set up a specific version of model so for example, if you extend the order model it can be set as order_type. It can be powerful as a way to extend package but in our case, it might bring more confusion than benefits so quickly renaming it to `foreignIdFor`.